queries/tcl/indents.scm

```diff
@@ -0,0 +1,24 @@
+[
+  (braced_word_simple)
+  (namespace)
+  (command)
+  (conditional)
+  (foreach)
+  (while)
+  (try)
+  (procedure)
+  (command_substitution)
+] @indent.begin
+
+(quoted_word) @indent.align
+
+([
+  (expr)
+  (arguments)
+] @indent.align
+  (#set! "delimiter" "{}"))
```

Comment:
```suggestion
  (#set! "indent.open_delimiter" "{")
  (#set! "indent.close_delimiter" "}")
```

Reply:
Do I have to submit this to tree-sitter-grammars :thinking: If yes then I'll file a PR there

Reply:
I think it's best to get it in shape here, then upstream _en bloc_. I also needed to make highlight query changes.

Reply:
I'm half wanting to upstream the queries here. The grammar explicitly states that it doesn't require `nvim-treesitter`, and `@indent`, `@branch` is somewhat used by Zed/Helix. Meaning enforcing nvim-treesitter indent queries might not be needed.